examples/ext/ext.s

```diff
@@ -0,0 +1,7 @@
+// Code generated by command: go run asm.go -out ext.s. DO NOT EDIT.
+
+// func StructFieldB(e github.com/mmcloughlin/avo/examples/ext/ext.Struct) byte
```

Comment:
Reminder to look into how to avoid printing the entire import path for the external type.